sale_order_lot_selection/models/stock_lot.py

```
@@ -0,0 +1,48 @@
+# Copyright (C) 2025 Cetmix OÜ
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import _, models
+from odoo.exceptions import AccessError
+
+
+class StockLot(models.Model):
+    _inherit = "stock.lot"
+
+    def action_add_to_sale_order(self):
```

Ruff (0.8.2)

Missing return type annotation for public function `action_add_to_sale_order`

(ANN201)